src/components/ResultsItemDetails.vue

```diff
@@ -61,8 +60,8 @@ export default {
     }
   },
   methods: {
-    favoritePark(parkName) {
-      eventBus.$emit('savePark', parkName);
+    savePark() {
+      this.$store.commit('savePark', this.parkName)
```

Comment:
`commit` is the way a mutation is triggered, and its arguments include the mutation (method) that is being invoked, as well as the payload (if any)

Reply:
Interesting, thanks for making note of that.